src/distilabel/steps/task/quality_scorer.py

```diff
@@ -75,21 +77,22 @@ def format_input(self, input: str) -> ChatType:  # type: ignore
     @property
     def outputs(self):
         """The output for the task is the `generation` and the `model_name`."""
-        return ["quality_score"]
+        return ["quality_scores"]
 
     def format_output(
         self, output: Union[str, None], input: Dict[str, Any]
     ) -> Dict[str, Any]:
-        """The output is formatted as a list with the rating of each instruction.
+        """The output is formatted as a list with the rating of each instruction-response tuple.
 
         Args:
             output (str): the raw output of the LLM.
```

Comment:
We decided not to include the type-hints within the docstrings, as those will be automatically inferred from the function/method signature, so feel free to remove those from the docstrings

```suggestion
            output: the raw output of the LLM.
```